js/binance.js

```diff
@@ -1475,6 +1475,67 @@ module.exports = class binance extends Exchange {
                 if (futuresSymbol in this.markets) {
                     return this.markets[futuresSymbol];
                 }
+            } else if (defaultType === 'option') {
+                // support expired option contracts
+                const settle = 'USDT';
+                if (!(symbol in this.markets)) {
```

Comment:
@Dan-krm we don't need this if, this branch already assumes `symbol` is not known